docs/_docs/setup.adoc

```diff
@@ -278,6 +278,49 @@ directory to discover other Ignite nodes.
 |=======================================================================
 
 
+== Setting JVM Options
+
+There are several ways you can set JVM options when starting a node with the `ignite.sh` script.
+These ways are described in the following sections.
+
+=== JVM_OPTS System Variable
+
+You can set the `JVM_OPTS` environment variable:
+
+[source, shell]
+----
+export JVM_OPTS="$JVM_OPTS -Xmx6G -DIGNITE_TO_STRING_INCLUDE_SENSITIVE=false"; $IGNITE_HOME/bin/ignite.sh
+----
+
+=== Command Line Arguments
+
+You can also pass JVM options by using the `-J` prefix:
+
+[source, shell]
+----
+./ignite.sh -J-Xmx6G -J-DIGNITE_TO_STRING_INCLUDE_SENSITIVE=false
```

Comment:
Could you add two tabs here for Unix and Windows respectively? The latter will run the same command but using "ignite.bat"

Reply:
done